docker-compose-win.yml

```diff
@@ -0,0 +1,56 @@
+# before run use command `docker volume create postgres`
+# run docker-compose -f docker-compose-win.yml up
+version: "3"
+services:
+  web:
+    build: .
+    container_name: web01
+    env_file:
+      - ./.env
+    command: >
+            sh -c "python3 manage.py collectstatic --no-input &&
+            python3 manage.py makemigrations &&
+            python3 manage.py migrate &&
+            gunicorn config.wsgi -b 0.0.0.0:8000 --workers 3 --timeout 600"
+    volumes:
+      - .:/www
+    ports:
```

Comment:
Remove it, you don't need to open this port. At most you need to expose it.
The same everywhere except nginx.